app/helpers/steps_helper.rb

```diff
@@ -0,0 +1,9 @@
+module StepsHelper
```

Comment:
we could be more specific here, 
what about naming it `MeursingStepHelper`,
in case we add more step forms in the project ... for example `CalculateBananasStepForms`